lib/simplekiq/orchestration_job.rb

```diff
@@ -8,8 +8,20 @@ module OrchestrationJob
 
     extend Forwardable
     def_delegators :orchestration, :run, :in_parallel
+    attr_reader :orchestration
+
+    def self.included(klass)
+      klass.extend ClassMethods
+    end
+
+    module ClassMethods
+      def child_job_options(*args)
```

Comment:
Curious why you made this a class method? It's not technically stateful, but it's operating on `args` so it feels like it belongs with `perform`.

You could argue that `perform` should also be a class method, which I might agree with, but it's not the convention that Sidekiq has gone with.